packages/calcite-components/src/components/tab-nav/tab-nav.e2e.ts

```diff
@@ -47,7 +46,7 @@ describe("calcite-tab-nav", () => {
 
     await page.keyboard.press("Enter");
     await page.waitForChanges();
-    expect(activeEventSpy).toHaveReceivedEventTimes(2);
+    expect(activeEventSpy).toHaveReceivedEventTimes(1);
```

Comment:
Can you update the test to cover emitting for both keyboard and mouse interactions? The current version won’t fail if one of the two interactions stops emitting.